project_national/housing_characteristics/Geometry Attic Type.tsv

```diff
@@ -1202,5 +1202,5 @@ West South Central	Single-Family Detached	4+	<1940	0	1	0	0	11	0	0
 # Created by: sources\recs\recs2020\tsv_maker.py
 # Description: The dwelling unit attic type.
 # Source: Constructed using U.S. EIA 2020 Residential Energy Consumption Survey (RECS) microdata.
-# Assumption: Multi-Family building types have Flat Roof (None) only.
+# Assumption: Multi-Family building types and Mobile Homes have Flat Roof (None) only.
```

Comment:
Looks like we are changing some of the logic/assumptions around mobile homes? Anything here related to https://github.com/NREL/resstock/issues/1098?

Reply:
This has been the assumption from when the major RECS2020 update was made (hence no tsv change here). We just forgot to list the assumption, which is what's being updated here.

I am not quite sure what belly is referring to, but with a flat roof, there can never be ducts in the "attic" for mobile/manufactured homes.